bootstrap

```diff
@@ -171,13 +171,7 @@ while (true) {
   }
 
   if (strlen($body) > 0) {
-    if($event['httpMethod'] === 'POST'){
-      curl_setopt($ch, CURLOPT_POSTFIELDS, $body);
-    }    
-    curl_setopt($ch, CURLOPT_INFILESIZE, strlen($body));
-    curl_setopt($ch, CURLOPT_READFUNCTION, function ($ch, $fd, $length) use ($body) {
-      return $body;
-    });
+    curl_setopt($ch, CURLOPT_POSTFIELDS, $body);
```

Comment:
Why do we no longer need `CURLOPT_INFILESIZE` and `CURLOPT_READFUNCTION`?

Reply:
The `CURLOPT_READFUNCTION` callback is only called if there was a file stream specified in `CURLOPT_INFILE`, so it wasn't being called in this instance.